src/resources/lib/modules/services.py

```diff
@@ -305,7 +305,7 @@ def load_values(self):
             # SAMBA
 
             if os.path.isfile(self.SAMBA_NMDB) and os.path.isfile(self.SAMBA_SMDB):
-                self.struct['samba']['settings']['samba_autostart']['value'] = self.oe.get_service_state('samba')
+                self.struct['samba']['settings']['samba_autostart']['value'] = "1"
```

Comment:
It doesn't make any difference to Python in this case, but the convention in this file is to use apostrophes for strings, would you mind changing `"1"` to `'1'`?